src/branchUniversalObject.js

```diff
@@ -66,7 +66,7 @@ export default async function createBranchUniversalObject(identifier, options =
 
       return this._tryFunction(RNBranch.showShareSheet, shareOptions, linkProperties, controlParams)
     },
-    // deprecated in favor of userCompletedAction(RegisterViewEvent)
+    // deprecated in favor of BranchEvent
     registerView() {
```

Comment:
Does this comment imply we should remove `registerView` as well? It's just an alias to a `BranchEvent.logEvent` underneath

Reply:
On the iOS side, I believe there is still some code that works with Spotlight so I don't think we can fully remove it yet.